test/inductor/test_torchinductor_opinfo.py

```diff
@@ -186,11 +186,11 @@ def process(device_type):
     "nn.functional.avg_pool2d": {i64},
     "nn.functional.adaptive_avg_pool2d": {f16},
     "nn.functional.ctc_loss": {f32, f64},
-    "nn.functional.gaussian_nll_loss": {f32, f64},
+    "nn.functional.gaussian_nll_loss": {f16, f32, f64},
     "nn.functional.local_response_norm": {i64},
     "nn.functional.one_hot": {i64},
     "nn.functional.rrelu": {f32, f64},
-    "nn.functional.triplet_margin_with_distance_loss": {f32, f64, i32, i64},
```

Comment:
Regarding this operation and `gaussian_nll_loss`, the reason why we add `fp16` is that the two operations support FP16 now.